### PR TITLE
Update ARIA11 to address issue #1308

### PR DESCRIPTION
**Pull request by fstrr**
Adds text for ARIA 1.2 form role.

Closes https://github.com/w3c/wcag/issues/1308

### CODE REVIEW

File: techniques/aria/ARIA11.html [hunk 1]
```diff
@@ -26,7 +26,7 @@ <h1>Using ARIA landmarks to identify regions of a page</h1>
...
          <li><code class="language-html">main</code>: A region that contains a page's main content.</li>
          <li><code class="language-html">region</code>: A region that contains a perceivable section of the page containing content that is sufficiently important for users to be able to navigate to the section. A <code class="language-html">region</code> landmark isn't exposed as a landmark region unless it has an accessible name.</li>
-         <li><code class="language-html">form</code>: A region of the document that represents a collection of form-associated elements, some of which can represent editable values that can be submitted to a server for processing.</li>
+         <li><code class="language-html">form</code>: A region of the document that represents a collection of form-associated elements, some of which can represent editable values that can be submitted to a server for processing. It maps to <code class="language-html">form</code> in HTML, and needs an accessible name to be exposed as a landmark.</li>
```
Comment (scottaohara) [inline techniques/aria/ARIA11.html:29]: not against the idea of mentioning what element this maps to in html - but now this is the only one that does.

there are a bunch of other nits i have with this doc (outside of the scope of this PR) - would it be ok to rectify those with this PR, or do we need to merge this and then i make another PR to make those corrections?

Comment (fstrr) [reply techniques/aria/ARIA11.html:29]: Depends! I'm increasingly not a fan of PRs that wander off topic, but if it's just a couple of minor things then we could add them. I could also back out the mapping content to put in another PR if your bunch of nits is large.

**Review by scottaohara — APPROVED** [review]
changing my review to an approval - since any issues i have with this doc are out of scope for this PR
